about-me.html

```diff
@@ -0,0 +1,142 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8"> 
```

Comment:
your missing the responsive meta tag here, look at the curriculum under responsive stuff, theres a meta tag you need to add to your head